src/createServer.js

```diff
@@ -1,3 +1,62 @@
-// Write code here
-// Also, you can create additional files in the src folder
-// and import (require) them here
+const http = require('http');
+const { convertToCase } = require('./convertToCase/convertToCase');
+
+const availableCases = ['SNAKE', 'KEBAB', 'UPPER', 'CAMEL', 'PASCAL'];
+
+function createServer() {
+  return http.createServer((req, res) => {
+    const url = new URL(`http://${req.headers.host}${req.url}`);
+    const targetCase = url.searchParams.get('toCase');
+    const originalText = url.pathname.slice(1);
+
+    const isError = !originalText
+      || !targetCase
+      || !availableCases.includes(targetCase);
```

Comment:
You already have these checks below, don't duplicate them, just check for errors.length to send an error
```suggestion
```